galaxy_ng/app/urls.py

```diff
@@ -18,6 +18,10 @@
     urlpatterns as resource_api_urls,
 )
 
+from ansible_base.rbac.urls import (
+    api_version_urls as dab_rbac_urls,
```

Comment:
It's really all DAB URLs. This might add more endpoints than you're not expecting, but that is intended on the DAB side.